docs/src/tut/mkt/index.md

```diff
@@ -0,0 +1,1323 @@
+load: /examples/mkt-2-basic/index.rsh
+range: 27 - 33
+```
+
+* Lines 27 to 33: `sellerInteract`.
+ This will hold all properties accessible by the `seller`.
+
+* Lines 28 to 31: `sellerInfo`.
+ This is an `Object` that will contain the `announcement` and `products` that were declared on lines 5 and 13 respectively.
+
+* Line 32: `reportReady`.
+ This is a `{!rsh} function` that takes in the `announcement` and `products` and returns nothing.
+
+
+Clear this code for the `buyerInteract` `{!rsh} Object`:
+
+```reach
+load: /examples/mkt-1-starter/index.rsh
+range: 7 - 7
+```
+
+Type in this code instead:
+
+```reach
+load: /examples/mkt-2-basic/index.rsh
+range: 35 - 37
+```
+
+* Lines 35 to 37: `buyerInteract`.
+This will hold all properties accessible by the `buyer`. 
+
+* Line 36: `shop`.
+ This is a `{!rsh} function` that represents the buyer's process in looking through the `seller`'s `products` and deciding which one to pick and how many are needed.
+ It takes in an `Object` of the `seller`'s `announcement` and `products`. 
+ And then returns an `Object` containing the `buyer`'s `choice` and `quantity`.
+
+That is all that needs to be done in the backend for now. 
+Attention will be given to the frontend now.
+
+### {#tut-mkt-basic-accounts} Create Accounts
+In the [previous section](##tut-mkt-starter-frontend), contracts were setup for the `{!rsh} Participant`s:
+
+```reach
+load: /examples/mkt-1-starter/index.mjs
+range: 5 - 6
+```
+
+:::note
+A contract is the same as a written agreement that must be signed by all `{!rsh} Participant`s before any transaction can take place.
+
+Deploying a contract is the act of creating the terms of participation. 
+This is done by the `{!rsh} Participant` who initiates the transaction.
+
+Attaching to a contract is agreeing to the terms of participation. 
+It is just like signing a physical contract paper that was handed over to you.
+
+This project will have the `seller` deploy the contract while the `buyer` will attach to it.
+:::
+
+These contracts require that each of the `{!rsh} Participant`s have an account set up before entering the contract.
+
+:::note
+Having an account is like having a book that keeps track of the activities of a particular user. 
+It holds information such as name, remaining money, and history of transaction.
+The DApp uses that record to manage the user while using the application.
+
+For example, a user's account balance would help the DApp to decide if the user is eligible to complete a transaction or not
+:::
+
+Just before the contracts, type the following code:
+
+```reach
+load: /examples/mkt-2-basic/index.mjs
+range: 5 - 8
+```
+
+* Line 5 sets up some funds that can be used by each `{!rsh} Participant`s as starting balance.
+`{!rsh} parseCurrency` converts the figure passed into it to an acceptable currency on a DApp.
+
+:::note
+`{!rsh} parseCurrency` `{!rsh} function` is an in-built method for converting regular currencies into currencies acceptable on a DApp.
```

Comment:
lines 318 and 321 basically say the same thing. Also, the phrasing of both could use work to be clearer.